prow/config/config.go

```diff
@@ -507,6 +537,51 @@ func validatePresets(name string, labels map[string]string, spec *v1.PodSpec, pr
 	return nil
 }
 
+func validatePodSpec(name string, jobType kube.ProwJobType, spec *v1.PodSpec) error {
+	if spec == nil {
+		return nil
+	}
+
+	if len(spec.InitContainers) != 0 {
+		return fmt.Errorf("job %s specified init containers, which is not allowed", name)
```

Comment:
Should jobs be allowed to specify init containers if they are not using decoration?

Reply:
Not today -- we need to have a strict paradigm for this sort of thing. It's not supported today, unless we come up with a strict design that allows for it to happen, seems OK to continue to not allow it.

Reply:
Hi Steve, does "a strict paradigm for this sort of thing" mean the order of initContainers?